### PR TITLE
GPT-2 Plugin

### PR DESCRIPTION
**Pull request by frazerk**

### CODE REVIEW

File: plugins/gpt2.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import os
```
Comment (evpotter) [inline plugins/gpt2.py:1]: this file has too much python

**Review by evpotter — CHANGES_REQUESTED** [review]

File: plugins/gpt2.py (hunk 1 above)
Comment (frazerk) [reply plugins/gpt2.py:1]: oh dang

**Comment by evpotter** [comment]
I also feel there should be a description about what the PR does. At least more than "GPT-2 Plugin". Also commit messages should follow more closely with this: [Git Commits](https://chris.beams.io/posts/git-commit/) @frazerk

**Comment by frazerk** [comment]
I can be more descriptive in the future, but I figured it was unnecessary since a) this is a silly chatbot and b) you guys had already seen this code in action.

**Comment by evpotter** [comment]
I'm just being an asshole lol I guess my joke didn't hit :(

**Comment by frazerk** [comment]
Please remove the Santa Hat from the VS Code logo

**Review by evpotter — APPROVED** [review]

**Pull request merged**